src/shared/dropdown-list-v2/nested-dropdown-list-helpers.ts

```diff
@@ -230,6 +222,7 @@ export const updateSelectedState = <T>(
             }
         }
     });
+    return res;
 };
 
 export const findIndexFromStart = <T>(
```

Comment:
`findIndexFromStart` and `findIndexFromEnd`  can be removed if they're no longer in use